functions/openhab.js

```diff
@@ -401,10 +480,12 @@ function syncAndDiscoverDevices(token, success, failure) {
 
 	// Checks for a Fahrenheit tag and sets the righ property on the
 	// attributeDetails response object
-	var setTempFormat = function(item, attributeDetails){
+	var setTempFormat = function(item, attributeDetails) {
 		if (item.tags.indexOf('Fahrenheit') > -1 || item.tags.indexOf('fahrenheit') > -1) {
+			attributeDetails.availableThermostatModes = 'off, cool, heat, on, heatcool';
```

Comment:
This string should be moved to some constant, 
It's also declared twice, so you might want to assign it above the `if` condition